### DISCUSSION TITLE
Fault bucket type 4 when resuming from Hibernation

### DISCUSSION THREAD

**Discussion by ghost** [audacity/audacity#4237] [Q&A] [answered]
### Bug description

In Audacity 3.3.0-alpha20230104, the application stops responding and crashes when resuming from hibernation.
This is the message that appears in Event Viewer
Fault bucket 1363678673772541818, type 4
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: Audacity.exe
P2: 3.3.0.0
P3: 63b5818e
P4: ntdll.dll
P5: 10.0.22621.900
P6: a97a9ed6
P7: c0000006
P8: 00000000000209e2
P9: 
P10: 

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.bdfabb5f-e026-43c6-9b93-7835039944cb.tmp.mdmp
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.2a89f75d-1c5d-41a7-8377-5b470cd3ba58.tmp.WERInternalMetadata.xml
WPR_initiated_DiagTrackMiniLogger_OneTrace_User_Logger_20230123_1_EC_0_inject.etl
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.20202a98-5a9b-4306-be2a-bdf72c50af5b.tmp.etl
WPR_initiated_DiagTrackMiniLogger_WPR System Collector_inject.etl
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.f35bc3a0-a0d6-49d0-8fc0-a362d1bdd3a7.tmp.etl
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.e24c8597-44a6-4c62-86ef-656399e953b4.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.2747aa5d-a47d-4aa4-ac89-d317863897a0.tmp.txt
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.0adbdca3-f28e-46a5-8380-2452b6c2e39f.tmp.xml

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_Audacity.exe_2846f9b313cdb8182bc26ce7c5c66242e8b47a0_4854c3e6_fef8e8a5-dfaa-458c-bb88-f53ce74656fd

Analysis symbol: 
Rechecking for solution: 0
Report Id: 96c4d637-dfe2-4951-9b2d-a4013ba2d79d
Report Status: 268435456
Hashed bucket: e9e8aa243ce9b1fa72ecc28759fcf77a
Cab Guid: 0
Windows cannot access the file  for one of the following reasons: there is a problem with the network connection, the disk that the file is stored on, or the storage drivers installed on this computer; or the disk is missing. Windows closed the program Audacity® Cross-Platform Sound Editor because of this error.

Program: Audacity® Cross-Platform Sound Editor
File: 

The error value is listed in the Additional Data section.
User Action
1. Open the file again. This situation might be a temporary problem that corrects itself when the program runs again.
2. If the file still cannot be accessed and
	- It is on the network, your network administrator should verify that there is not a problem with the network and that the server can be contacted.
	- It is on a removable disk, for example, a floppy disk or CD-ROM, verify that the disk is fully inserted into the computer.
3. Check and repair the file system by running CHKDSK. To run CHKDSK, click Start, click Run, type CMD, and then click OK. At the command prompt, type CHKDSK /F, and then press ENTER.
4. If the problem persists, restore the file from a backup copy.
5. Determine whether other files on the same disk can be opened. If not, the disk might be damaged. If it is a hard disk, contact your administrator or computer hardware vendor for further assistance.

Additional Data
Error value: 0xC000026E
Disk type: 0x0
Faulting application name: Audacity.exe, version: 3.3.0.0, time stamp: 0x63b5818e
Faulting module name: ntdll.dll, version: 10.0.22621.900, time stamp: 0xa97a9ed6
Exception code: 0xc0000006
Fault offset: 0x00000000000209e2
Faulting process id: 0x0x1E40
Faulting application start time: 0x0x1D930DD7275CA8B
Faulting application path: D:\Other Files\Folders\aud\Audacity.exe
Faulting module path: C:\WINDOWS\SYSTEM32\ntdll.dll
Report Id: 96c4d637-dfe2-4951-9b2d-a4013ba2d79d
Faulting package full name: 
Faulting package-relative application ID: 

### Steps to reproduce

1. turn off computer by hibernating it
2. turn it back on
3. after a few minutes it will crash without warning.

### Expected behavior

Not crash when it does this, especially via a flash drive!

### Actual behavior

Event log writes data immediately after the program closes.

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

**Comment by LWinterberg** [comment, marked as answer]
It seems like your USB port gets disabled while hibernating and isn't back online when Windows is trying to load the program back.  There may be a fix to this deep into either the Windows or the BIOS power settings, or your device may have an always-on "phone charging" USB port which is available earlier in the boot process.

**Reply by ghost** [reply]
Looks like this may be the case. I will move the audacity files off the flash drive and get it moved to the C drive.

**Comment by ghost** [comment]
Its doing it again.
Fault bucket , type 0
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: Audacity.exe
P2: 0.0.0.0
P3: 63b5818e
P4: ntdll.dll
P5: 10.0.22621.900
P6: a97a9ed6
P7: c0000005
P8: 00000000000a3db7
P9: 
P10: 

Attached files:
\\?\C:\Users\snowc\AppData\Local\Temp\WER.4d80e11f-b477-41d7-896d-a587a3e16830.tmp.WERDataCollectionStatus.txt
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.084e0787-48a7-456b-b432-981cc82e9622.tmp.WERInternalMetadata.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.eb3a654a-eb1b-4569-8577-e8e9e5e46758.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.51abf723-1646-4bcc-bef1-9d081f09d871.tmp.txt
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.923c4244-748b-47e3-a57e-dc16c0c59a71.tmp.xml
\\?\C:\Users\snowc\AppData\Local\Temp\WER.e8601ef4-8ba2-4835-91d6-51019a71787a.tmp.appcompat.txt

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportQueue\AppCrash_Audacity.exe_39ba8dacafaedc9e578df791a667426811e1b75_2d6f6458_cab_3393188c-d0e8-4f47-994e-d0b623819637

Analysis symbol: 
Rechecking for solution: 0
Report Id: 3ee18cd4-58a7-4686-801a-60697e7854cf
Report Status: 100
Hashed bucket: 
Cab Guid: 0
